cmd/lncli/walletrpc_active.go

```diff
@@ -265,15 +265,18 @@ func bumpFee(ctx *cli.Context) error {
 }
 
 var bumpCloseFeeCommand = cli.Command{
-	Name:      "bumpclosefee",
-	Usage:     "Bumps the fee of a channel closing transaction.",
+	Name:      "bumpforceclosefee",
```

Comment:
To not break existing scripts, should we keep the old command but set it to `Hidden: true` and mention to use the new one instead?

Reply:
yeah we need to use the hidden flag to keep the old cli in case it breaks people's scripts.

Reply:
or we keep the old command, and just keep the new description to avoid bloating the code base ?